vibi-dpu/src/core/coverage.rs

```diff
@@ -14,24 +15,32 @@ pub async fn process_coverage(hunkmap: &HunkMap, review: &Review) {
     for prhunk in hunkmap.prhunkvec() {
         // calculate number of hunks for each userid
         let coverage_map = calculate_coverage(&hunkmap.repo_owner(), prhunk);
+        let coverage_cond = !coverage_map.is_empty();
+        println!("!coverage_map.is_empty() = {:?}", &coverage_cond);
+        println!("repo_config.comment() = {:?}", repo_config.comment());
+        println!("repo_config.auto_assign() = {:?}", repo_config.auto_assign());
         if !coverage_map.is_empty() {
-            // get user for each user id
-            // add reviewers
-            let mut author_set: HashSet<String> = HashSet::new();
-            author_set.insert(prhunk.author().to_string());
-            for blame in prhunk.blamevec() {
-                if author_set.contains(blame.author()) {
-                    continue;
+            if repo_config.comment() {
+                println!("Inserting comment...");
```

Comment:
Add a tag for the log so that we can identify which pull request (and which repo) this log corresponds to.